src/Stratis.Bitcoin.Features.Consensus/CoinViews/CoinView.cs

```diff
@@ -57,8 +57,9 @@ public interface ICoinView
         /// and restoring recently spent outputs as UTXOs.
         /// </para>
         /// </summary>
+        /// <param name="targetHeight">The target height. This is a hint and may be ignored by the implementation in favor of rewinding a single block at a time.</param>
         /// <returns>Hash of the block header which is now the tip of the rewound coinview.</returns>
-        uint256 Rewind();
+        uint256 Rewind(int? targetHeight = null);
```

Comment:
See comments on overloading and/or making this parameter mandatory.

Reply:
Made mandatory.